pom.xml

```diff
@@ -32,6 +32,7 @@
     <inceptionYear>2016</inceptionYear>
 	<properties>
 		<main-class>nl.knaw.dans.easy.license.app.Command</main-class>
+		<easy.emd.version>3.0</easy.emd.version>
```

Comment:
@lindareijnhoudt linked to maven's [release-plugin](http://maven.apache.org/maven-release/maven-release-plugin/update-versions-mojo.html) which in turn links to [update versions](http://maven.apache.org/maven-release/maven-release-plugin/examples/update-versions.html) but I still don't get whether to execute the latter command for all projects using the emd/ddm, the easy-build project or whatever.